solutions/cses-1639.mdx

```diff
@@ -2,9 +2,71 @@
 id: cses-1639
 source: CSES
 title: Edit Distance
-author: x
+author: Sofia Yang
 ---
 
 [Editorial (icecuber)](https://codeforces.com/blog/entry/70018)
 
 [CPH 7.5 (Edit Distance)](https://usaco.guide/CPH.pdf#page=84)
+
+# Implementation
+**Time Complexity:** $\mathcal{O}(NM)$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class editDistance {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio();
+		char[] arr1 = io.next().toCharArray();
+		char[] arr2 = io.next().toCharArray();
+		int N = arr1.length;
+		int M = arr2.length;
```

Comment:
ok (possibly) better var names
str1, str2, str1Len, str2Len
even better just do away w/ the N and M and instead directly access the `length` attribute each time